pkg/kubelet/dockershim/docker_container.go

```diff
@@ -131,6 +131,10 @@ func (ds *dockerService) CreateContainer(podSandboxID string, config *runtimeapi
 			OpenStdin: config.Stdin,
 			StdinOnce: config.StdinOnce,
 			Tty:       config.Tty,
+			// Disable Docker's health check until we officially support it.
+			Healthcheck: &dockercontainer.HealthConfig{
```

Comment:
Unfortunately, the docker-related code changed *a lot* in the past 3 releases. Updating the docker client for each release will be a completely different PR....
I think it might be too intrusive. Let's think it through first...